lxc-binding.c

```diff
@@ -44,6 +44,14 @@ pid_t go_lxc_init_pid(struct lxc_container *c) {
 	return c->init_pid(c);
 }
 
+int go_lxc_init_pidfd(struct lxc_container *c) {
+	return c->init_pidfd(c);
+}
+
+int go_lxc_seccomp_notify_fd(struct lxc_container *c) {
+	return c->seccomp_notify_fd(c);
+}
```

Comment:
Don't we need to do something so we can still build on older liblxc?

Reply:
I hadn't seen anything for other extensions so far but criu. I'll take another look.

Reply:
It doesn't look like we're doing anything for ConsoleLog and friends...

Reply:
Yeah, I think we simply don't don anything...

Reply:
Hmm, that's a bit unfortunate, can we do something for this one since we know what release it's in?

I'm a bit worried of breaking existing users that aren't on 4.0 yet.

Reply:
Fixed. I looked at the wrong macro before with case sensitivity. :)
We do in fact special case new features under `VERSION_AT_LEAST`. I was only grepping for `Version`. :)